docker-run-facebook-to-engagement-db.sh

```diff
@@ -0,0 +1,72 @@
+#!/bin/bash
+
+set -e
+
+PROJECT_NAME="$(<configurations/docker_image_project_name.txt)"
+IMAGE_NAME=$PROJECT_NAME-facebook-to-engagement-db
+
+while [[ $# -gt 0 ]]; do
+    case "$1" in
+        --incremental-cache-volume)
+            INCREMENTAL_ARG="--incremental-cache-path /cache"
+            INCREMENTAL_CACHE_VOLUME_NAME="$2"
+            shift 2;;
+        --)
+            shift
+            break;;
+        *)
+            break;;
+    esac
+done
+
+# Check that the correct number of arguments were provided.
+if [[ $# -ne 4 ]]; then
+    echo "Usage: $0 
+    [--incremental-cache-volume <incremental-cache-volume>] 
+    <user> <google-cloud-credentials-file-path> <configuration-module> <data-dir>"
```

Comment:
Can you align these args with the recently merged https://github.com/AfricasVoices/Test-Pipeline-Engagement-DB/pull/159? i.e. copy in configuration and code schemes rather than requiring them at build time

Reply:
updated thanks.